packages/create-svelte/bin.js

```diff
@@ -75,8 +76,16 @@ async function main() {
 					message: 'Add type checking?',
 					initial: false,
 					choices: [
-						{ title: 'Type-checked JavaScript', value: 'checkjs' },
-						{ title: 'TypeScript', value: 'typescript' },
+						{
+							title: 'JavaScript with checkJs',
+							value: 'checkjs',
+							description: 'using JavaScript comments for types.'
+						},
+						{
+							title: 'TypeScript',
+							value: 'typescript',
+							description: 'using TypeScript annotations for types.'
+						},
 						{ title: 'None', value: null }
```

Comment:
maybe add description here too? For `None` description can be `Standard JS without any type checking` or something like that.

Reply:
None is fairly self-descriptive, don't think we need to add verbiage to it

Reply:
I agree with @Mlocik97 here. 
It may seem a bit redundant, but it makes it easier to compare the options, and it confirms what the user is thinking.